app/views/search/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= search_form_for @q do |f| %>
   <div class="form-group">
     <%= f.label t("exercise_search") %>
-    <input class="form-control" id="q_language_name_or_title_or_description_or_locale_cont" name="q[language_name_or_title_or_description_or_locale_cont]" type="search">
+    <input class="form-control" id="q_language_name_tags_name_or_title_or_description_or_locale_cont" name="q[language_name_or_tags_name_or_title_or_description_or_locale_cont]" type="search">
```

Comment:
¿Is there any way to use a shorter but more expressive name like `query` here? It is really error prone to have such long name. 

Actually, this PR is failling because of a broken test related with that rename